src/pages/EventAdministration/components/EventFormAdmin.tsx

```diff
@@ -25,21 +105,68 @@ const EventFormAdmin = ({ eventId }: EventFormAdminProps) => {
 
   const EventFormEditor = ({ formType }: EventFormEditorProps) => {
     const createForm = useCreateForm();
+    const showSnackbar = useSnackbar();
+    const { data: formtemplates = [] } = useFormTemplates();
+    const [isCreatingForm, setIsCreatingTemplate] = useState(false);
+    const [formtemplateId, setFormtemplateId] = useState('-');
+    const { data: form, isLoading, isError } = useFormById(formtemplateId);
 
-    const newForm: EventFormCreate = {
+    const newForm: FormCreate = {
       title: `${event.title} - ${formType === EventFormType.SURVEY ? 'påmeldingsskjema' : 'evalueringsskjema'}`,
       type: formType,
       event: event.id,
       resource_type: FormResourceType.EVENT_FORM,
       fields: [],
     };
 
+    const newFormTemplateCreate: FormCreate = {
+      title: 'BLANK',
+      resource_type: FormResourceType.FORM,
+      fields: [],
+      viewer_has_answered: false,
+      template: true,
+    };
+
     const onCreate = async () => createForm.mutate(newForm);
 
+    const onCreateTemplate = async () => {
+      createForm.mutate(newFormTemplateCreate, {
+        onSuccess: (data) => {
+          showSnackbar(data.title, 'success');
+          setFormtemplateId(data.id);
+          setIsCreatingTemplate(true);
+        },
+        onError: (e) => {
+          showSnackbar(e.detail, 'error');
+        },
+      });
+    };
+
     return (
-      <Button fullWidth onClick={onCreate} variant='outlined'>
-        Opprett {formType === EventFormType.SURVEY ? 'påmeldingsskjema' : 'evalueringsskjema'}
-      </Button>
+      <>
+        <Button fullWidth onClick={onCreate} variant='outlined'>
+          Opprett {formType === EventFormType.SURVEY ? 'påmeldingsskjema' : 'evalueringsskjema'}
+        </Button>
+        <Expand flat header='Bruk en mal' sx={{ mt: 1 }}>
+          {isCreatingForm ? (
+            <>
+              {!isLoading && !isError && form && (
+                <FormFieldsEditor canEditTitle={true} form={form} onSave={() => setIsCreatingTemplate(false)}></FormFieldsEditor>
+              )}
+            </>
+          ) : (
```

Comment:
Tror dette bør funke
```suggestion
          {isCreatingForm ? (!isLoading && !isError && form && (
                <FormFieldsEditor canEditTitle={true} form={form} onSave={() => setIsCreatingTemplate(false)} />
              )) : (
```

Reply:
Fikk ikke til å skrive det sånn. TS klagde på type error